src/pages/Facility/settings/productKnowledge/ProductKnowledgeList.tsx

```diff
@@ -66,19 +66,20 @@ function ProductKnowledgeCard({
               </p>
             )}
           </div>
-          <Button
-            variant="outline"
-            size="sm"
-            onClick={() =>
-              navigate(
-                `/facility/${facilityId}/settings/product_knowledge/${product.id}`,
-              )
-            }
-          >
-            <CareIcon icon="l-edit" className="size-4" />
-            {t("see_details")}
-          </Button>
         </div>
+        <Button
+          variant="outline"
+          size="sm"
+          className="ml-auto block"
+          onClick={() =>
+            navigate(
+              `/facility/${facilityId}/settings/product_knowledge/${product.id}`,
+            )
+          }
+        >
+          <CareIcon icon="l-edit" className="size-4" />
+          {t("see_details")}
+        </Button>
```

Comment:
let's wrap with `Link` and use asChild for the button. 

Currently this does not allow users to right click and open in new tab and couple other things... All buttons with simply navigate onClick needs to be changed to button wrapped in link component ideally!

Reply:
mind sharing what's the error?